spikingjelly/activation_based/triton_kernel/flex_sn_inductor/kernel.py

```
@@ -1,205 +1,267 @@
-    bwd_str, bwd_name = generate_triton_code_str(bwd_graph, core_name + "_bwd")
-    if sum(mask) < expected:
-        shim_code, bwd_call_name = _make_bwd_shim(
-            bwd_name, n_saved, num_outputs, num_states, mask
-        )
-        bwd_str = bwd_str + shim_code
+    if _training_final_state_specialized_wins(info):
+        try:
+            fwd_final_state_kernel = get_flexsn_forward_final_state_kernel(
+                fwd_str, fwd_name, info=info
+            )
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)